lib/sidekiq_unique_jobs/web.rb

```diff
@@ -71,20 +71,20 @@ def self.registered(app) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
       end
 
       app.get "/locks/:digest" do
-        @digest = params[:digest]
+        @digest = h(params[:digest])
         @lock   = SidekiqUniqueJobs::Lock.new(@digest)
 
         erb(unique_template(:lock))
       end
 
       app.get "/locks/:digest/delete" do
-        digests.delete_by_digest(params[:digest])
-        expiring_digests.delete_by_digest(params[:digest])
+        digests.delete_by_digest(h(params[:digest]))
+        expiring_digests.delete_by_digest(h(params[:digest]))
         redirect_to :locks
       end
 
       app.get "/locks/:digest/jobs/:job_id/delete" do
-        @digest = params[:digest]
+        @digest = h(params[:digest])
         @lock   = SidekiqUniqueJobs::Lock.new(@digest)
         @lock.unlock(params[:job_id])
```

Comment:
@mhenrixon 
Both `params[:digest]` and `params[:job_id]` are derived from the request path, but only `params[:digest]` got the fix.  Is there a simple reason?

Reply:
This parameter is not displayed in the web UI, you're free to unlock by `<img onerror='whatever'>` if you like but its not going to do much. In a similar vein the changes to `/locks/:digest/delete` seem unnecessary.

Reply:
Thanks @Earlopain, so the fix is good, if a little overzealous.

When some are, and some are not, escaped, it might be good to document the reasons @mhenrixon, so people don't have to wonder, or become alarmed that there may be latent XSS issues here.

Reply:
To be clear then, only three endpoints were vulnerable *prior to this change*, and the changes here included some over protective escaping on some routes.

The vulnerable routes in the CVE are:

1. `/changelogs`
2. `/locks`
3. `/expiring_locks`

Reply:
Ah, must have opened the wrong view. I just looked at the usage in the view, as the internal usage on the ruby side isn't affected. 

Searched for @digest and escaped the parameter.

Reply:
That's great for consistency, and no one is load testing the admin UI, so the perf hit is immaterial, even if escaping the path-derived params isn't required.

Reply:
I'm almost done reporting the CVE!

Reply:
> ould it make sense to also escape the `job_id` for consistency though?

Absolutely, the idea was to escape everything but I honestly didn't see the job_id params.

I'll get that sorted and create a backport later today. I have to take my dog to the vet first.

Reply:
Sure thing @pboling, I just have to perform some self-care first. I should be available in an hour or so. 

Preferred means of communication?

Reply:
@mhenrixon I'll be in my zoom room for the next hours an a half.  Emailing you the link.